gettsim/taxes/favorability_check.py

```diff
@@ -14,74 +14,74 @@
 from gettsim.typing import IntSeries
 
 
-def beantrage_kind_freib_tu(
-    st_kein_kind_freib_tu: FloatSeries,
+def beantrage_kinder_fb_tu(
```

Comment:
Somewhat aside, but do I actually have to claim it or does the Finanzamt conduct the favorability check automatically? In the latter case, we should change the name.

Reply:
Rather the latter AFAIK.

`günstiger_kinderfreib_tu`?

Reply:
Almost -- I would prefer `kinderfreib_günstiger_tu`

Reply:
👍